src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Farmlands Portal Introduction Guide
+description: Farmlands Co-operative (Farmands) have partnered with Centrapay Payment Gateway (Centrapay) to deliver a secure web portal for Farmlands Card payments.
+nav:
+  path: Connections/Farmlands
+  title: Portal Introduction Guide
+  order: 2
```

Comment:
@IsaacMurtagh Worth checking in with @laurencemcknight on this order value - I don't know what will happen if 2 pages have the same order number right now. I don't think we have guardrails in place to detect this at build time (yet)

Reply:
Nice spot, I don't think it does anything, setting them both to one, but i'll set it to 3 for now.